### ISSUE TITLE
Create process for keeping synch with IANA IP address space databases

### ISSUE THREAD

**Issue by matsduf** [dotse/zonemaster-engine#301]
With PR #282 Zonemaster will mirror the IANA IP address space databases (1), but there is no process to keep them in sync. Should Zonemaster include a cron script that could be run regularily to keep the mirrored database with the IANA dito? Or should the synchronization only be done at installation time?

1) The databases:
http://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry-1.csv
http://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry-1.csv

**Comment by vlevigneron** [comment]
Yes this is the plan. The current code must be considered as a first step of a more complex process we need to describe and put in place to keep the 2 files synchronized with the IANA database.

**Comment by pawal** [comment]
Including a script in the *source code* that optionally could be run as a cron job sounds good. Since the engine is a library it should not install scripts, imho. Especially since a typical installation will set these files as owned by root.

**Comment by sandoche2k** [comment]
@vlevigneron can you have a look

**Comment by sandoche2k** [comment]
@sandoche2k move to 2018

**Comment by sandoche2k** [comment]
@vlevigneron do you think it is possible to fix for 2018.1?

**Comment by sandoche2k** [comment]
The fix is made for 384
